tripal/tripal
# Sequence length + Md5sum fields

#

Issue #1509


### Tripal Version: 4

## Description
Modified the tripal_chado tripal_collections YAML file to include new fields into the gene page bundle.



## Testing?
- To test the YAML you need to build a new docker on this branch
```
n=1509
git clone https://github.com/tripal/tripal tripal-$n
git checkout tv4g1-1509-sequence-fields
docker build --tag=tripaldocker:testing-$n --build-arg drupalversion="10.1.x-dev" --file tripaldocker/Dockerfile-php8.2-pgsql13 ./
docker run --publish=8082:80 -tid --name=${n} --volume=/home/admin/git/tripal-$n:/var/www/drupal9/web/modules/contrib/tripal --volume=/home/admin/docker_prepare:/usr/local/bin/docker_common tripaldocker:testing-$n
```
- Log in to http://localhost:8082
- Create an organism record
- Create a gene record and include a sequence e.g. `actg`
- The sequence should appear along with sequence length and sequence MD5 checksum
